routers/api/v2/tokens.go

```diff
@@ -0,0 +1,65 @@
+package v2
+
+import (
+	"github.com/gin-gonic/gin"
+	v2 "github.com/unicsmcr/hs_auth/authorization/v2"
+	"github.com/unicsmcr/hs_auth/routers/api/models"
+	"go.uber.org/zap"
+	"net/http"
+	"strings"
+)
+
+// POST: /api/v2/tokens/service
+// x-www-form-urlencoded
+// Request:	 owner string
+//			 allowedURIs string
+//			 expiresAt int64
+// Response: token string
+// Headers:  Authorization <- token
+func (r *apiV2Router) CreateServiceToken(ctx *gin.Context) {
+	var req struct {
+		Owner       string `form:"owner"`
+		AllowedURIs string `form:"allowedURIs"`
```

Comment:
```suggestion
		AllowedURIs []v2.UniformResourceIdentifier `form:"allowedURIs"`
```
Would this not work? We have a JSON unmarshaller for `UniformResourceIdentifier`, so I think it should work fine?

Reply:
I tried to do it that way, but since you can have multiple URIs in a comma-separated string, the unmarshaller can't handle that ATM

Reply:
Oh right, true. We could create custom binding logic and use [BindWith](https://godoc.org/github.com/gin-gonic/gin#Context.BindWith) but in this case it's probably not worth it.

Reply:
Yeah, I think in this case it may be simpler to use .Bind() to a string, it's something that could be added in the future